docs/ignore.md

```diff
@@ -26,6 +26,8 @@ It’s recommended to have a `.prettierignore` in your project! This way you can
 
 (See also the [`--ignore-path` CLI option](cli.md#--ignore-path).)
 
+By default pettier ignores what it will find in any .gitignore it finds in the current repository
```

Comment:
I don't think it's true, we don't read `.gitignore` file.

Reply:
That's what I thought that it looks for .gitignore files after I did quick dive into the code using github search.
I've stumbled across this DEFAULT_IGNORE in https://github.com/prettier/prettier/blob/529312e9acf8f48bebe67f7526cb629665b5a4f3/vendors/tempy.js#L5696

I might be wrong though... I don't have full day for that search

What I'm trying to highlight here is that I think it's natural question to ask "If prettier ignores some files by default". Because it does seem to not search files in .git or node_modules directories. So naturally you might ask yourself "Is there anything else it's ignoring out of the box?"

Just document that somehow please. What is the default ignoring rules if there are any?

Reply:
https://github.com/prettier/prettier/blob/cb997ca7abac937899bede96c3ff87aac16abc83/src/cli/expand-patterns.js#L48-L51

Reply:
Ok, there it is, thank you.
Sooo... Can't it be documented? Just mentioned that this is the case, that it ignores those version control directories and node_module when --with-node-modules is not specified?

It's not difficult to miss the fact that there is flag like --with-node-modules
and even when you find it in cli prettier --help it's doesn't carry information about ignoring .git, .svn and .hg...

Reply:
Sure!

Reply:
@fisker check PR now please